lib/sampler.py

```diff
@@ -80,6 +82,9 @@ def set_queue(self, queue, worker_id, worker_name):
         self._queue = queue
         self._watcher = None
 
+    def rss_summary(self):
```

Comment:
It would be nice to add a doc comment with description of return format.

Reply:
Added.